packages/scaffold/src/partials/w3m-input-token/index.ts

```diff
@@ -106,7 +111,19 @@ export class W3mInputToken extends LitElement {
 
   private onMaxClick() {
     if (this.token) {
-      SendController.setTokenAmount(Number(this.token?.quantity.numeric))
+      if (this.gasPriceInUSD) {
```

Comment:
If we don't have else state, maybe we could use both in same if?

```suggestion
      if (this.token && this.gasPriceInUSD) {
```